calcom/cal.com
# feat: Add more metadata to Stripe transaction

## What does this PR do?

This PR adds additional information to Stripe metadata and tweaks data added in a previous PR for better usability:
- **organizer id** and **username** - used to identify the cal user when multiple cal accounts are linked to the same Stripe account
- **event type title** instead of **event title** - allows aggregating across all like bookings by event type for total earnings by event type, the individual title is less useful
- **bookerName** - useful in interacting with user for charge disputes

Fixes https://github.com/calcom/cal.com/issues/10668, https://github.com/calcom/cal.com/issues/10678, https://github.com/calcom/cal.com/issues/8205

## Requirement/Documentation



- Original request was added [here](https://github.com/calcom/cal.com/issues/10668)

## Type of change


## How should this be tested?

- Have an organizer account with a linked stripe account offer a paid event type
- Create a booking of that paid event type
- Metadata should show in the Stripe transaction:

<img width="434" alt="Screenshot 2023-09-07 at 2 31 26 PM" src="https://github.com/calcom/cal.com/assets/2559359/948828a0-6b13-4ff9-95aa-1336c3e9af7e">

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
